apps/src/code-studio/pd/form_components_func/LabeledFormComponent.story.jsx

```diff
@@ -22,194 +22,192 @@ import {action} from '@storybook/addon-actions';
-          >
-            <LabeledRadioButtonsWithAdditionalTextFields
-              name="radioButtonsWithOther"
-              textFieldMap={{
-                [OTHER]: 'other',
-              }}
-            />
-          </Context>
-        ),
-      },
-      {
-        name: 'DynamicRadioButtonsWithAdditionalFields',
-        story: () => (
-          <Context
-            name="dynamicRadioButtonsWithOther"
-            label="Select a dynamic option and add text"
-          >
-            <LabeledDynamicRadioButtonsWithAdditionalTextFields
-              name="dynamicRadioButtonsWithOther"
-              options={[1, 2, 3]
-                .map(n => `Dynamic option #${n}`)
-                .concat([OTHER])}
-              textFieldMap={{[OTHER]: 'other'}}
-            />
-          </Context>
-        ),
-      },
-      {
-        name: 'DynamicCheckboxes',
-        story: () => (
-          <Context
-            name="dynamicCheckBoxes"
-            label="Select all dynamic options that apply"
-          >
-            <LabeledDynamicCheckBoxes
-              name="dynamicCheckBoxes"
-              options={[1, 2, 3].map(n => `Dynamic option #${n}`)}
-            />
-          </Context>
-        ),
-      },
-      {
-        name: 'DynamicCheckboxesWithAdditionalFields',
-        story: () => (
-          <Context
-            name="dynamicCheckboxesWithOther"
-            label="Select a dynamic option and add text"
-          >
-            <LabeledDynamicCheckBoxesWithAdditionalTextFields
-              name="dynamicCheckboxesWithOther"
-              options={[1, 2, 3]
-                .map(n => `Dynamic option #${n}`)
-                .concat([OTHER])}
-              textFieldMap={{[OTHER]: 'other'}}
-            />
-          </Context>
-        ),
-      },
-      {
-        name: 'Select',
-        story: () => (
-          <Context
-            name="select"
-            label="Select an option"
-            options={{
-              select: ['option 1', 'option 2', 'option 3'],
-            }}
-          >
-            <LabeledSelect name="select" placeholder="Select an option" />
-          </Context>
-        ),
-      },
-      {
-        name: 'Input',
-        story: () => (
-          <Context name="input" label="Enter some text">
-            <LabeledInput name="input" />
-          </Context>
-        ),
-      },
-      {
-        name: 'LargeInput',
-        story: () => (
-          <Context name="largeInput" label="Enter some longer text">
-            <LabeledLargeInput name="largeInput" />
-          </Context>
-        ),
-      },
-      {
-        name: 'UsPhoneNumber',
-        story: () => (
-          <Context name="usPhoneNumber" label="Enter a phone number">
-            <LabeledUsPhoneNumberInput name="usPhoneNumber" />
-          </Context>
-        ),
-      },
-    ]);
+export default {
+  title: 'LabeledFormComponents',
+  component: LabelsContext,
+  decorators: [reactBootstrapStoryDecorator],
```

Comment:
TIL how to keep using this... I've just been wrapping the stories in that div because it's only one line😂

Reply:
Me too! 😅 🤦‍♀️